.github/workflows/pr.yaml

```diff
@@ -1,59 +1,178 @@
+      # We need to replace the `C:/tmp` strings with `/tmp` throughout the files so that the incldues do not break
+      - run: find /tmp/windows_unzipped -type f -exec sed -i 's|C:/tmp|/tmp|g' {} +
+      - name: Download unix overlay and openssl files
+        uses: actions/download-artifact@v4
+        with:
+          name: 3.3.1.bcr.wip.unix.tar.gz
+          path: /tmp/unix/
+      - run: (cd /tmp && mkdir /tmp/unix_unzipped && tar -xzvf /tmp/unix/3.3.1.bcr.wip.unix.tar.gz -C /tmp/unix_unzipped && mkdir /tmp/unix_unzipped/linux && mkdir /tmp/unix_unzipped/mac)
+      - run: mkdir -p /tmp/unix_unzipped/linux/x86_64 && tar -xzvf /tmp/unix_unzipped/tmp/3.3.1.bcr.wip.linux-x86_64-clang.tar.gz -C /tmp/unix_unzipped/linux/x86_64
+      - run: mkdir -p /tmp/unix_unzipped/linux/arm64 && tar -xzvf /tmp/unix_unzipped/tmp/3.3.1.bcr.wip.linux-aarch64.tar.gz -C /tmp/unix_unzipped/linux/arm64
+      - run: mkdir -p /tmp/unix_unzipped/mac/x86_64 && tar -xzvf /tmp/unix_unzipped/tmp/3.3.1.bcr.wip.darwin64-x86_64-cc.tar.gz -C /tmp/unix_unzipped/mac/x86_64
+      - run: mkdir -p /tmp/unix_unzipped/mac/arm64 && tar -xzvf /tmp/unix_unzipped/tmp/3.3.1.bcr.wip.darwin64-arm64-cc.tar.gz -C /tmp/unix_unzipped/mac/arm64
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.13"
+      - run: ./download-buildifier.sh
+      - run: git config --global user.email "github@raccoons.build" && git config --global user.name "Raccoons Build"
+      - run: (cd bazel-central-registry && git remote add upstream https://github.com/bazelbuild/bazel-central-registry.git && git fetch upstream && git reset --hard upstream/main)
+      # We first run for all OS so we don't have any empty bzl files. But this run is just for Unix since we need to specify Unix paths for release_url_template
+      - run: python3 generate_combine_platforms.py --os=all --bcr_dir=$(pwd)/bazel-central-registry --overlay_tar_path=/tmp/3.3.1.bcr.wip.unix_merged.tar.gz --openssl_tar_path=/tmp/ --tag=3.3.1.bcr.wip --release_tar_url_template=file:///tmp/3.3.1.bcr.wip.unix_merged.tar.gz --buildifier=/tmp/buildifier.exe
+      - run: tar czf bcr.unix_merged.tar.gz bazel-central-registry
+      - name: Upload Unix overlay
+        uses: actions/upload-artifact@v4
+        with:
+          name: 3.3.1.bcr.wip.unix_merged.tar.gz
+          path: /tmp/3.3.1.bcr.wip.unix_merged.tar.gz
+          if-no-files-found: error
+      - name: Upload Unix bcr
+        uses: actions/upload-artifact@v4
+        with:
+          name: bcr.unix_merged.tar.gz
+          path: bcr.unix_merged.tar.gz
+          if-no-files-found: error
+      # Then we run for all OS but use it for Windows. This way we don't need to change any files at runtime in the test workflow. This is not an issue in prod because the release url template is a url not a local url.
```

Comment:
```suggestion
      # Then we run for all OS but use it for Windows. This way we don't need to change any files at runtime in the test workflow. This is not an issue in prod because the release url template is an https url not a local url.
```